doc/Contributions.md

```diff
@@ -56,6 +56,15 @@ Apache license 2.0
 !!! Info
     amoc stands for "A Murder of Crows"
 
+### amoc-arsenal-xmpp
+
+See [esl/amoc-arsenal-xmpp](https://github.com/esl/amoc-arsenal-xmpp) on GitHub for more details.
+
+A collection of scenarios for [amoc](#amoc), which we use to test MongooseIM.
+They can however be used to load test any XMPP server.
+
+Apache license 2.0
```

Comment:
Maybe make sense add link to `Apache license 2.0` https://www.apache.org/licenses/LICENSE-2.0 or maybe better to https://github.com/esl/amoc-arsenal-xmpp/blob/master/LICENSE

Reply:
Same for other places where used mention about license